rpc/backend/blocks.go

```diff
@@ -352,8 +365,8 @@ func (b *Backend) BlockBloom(blockRes *tmrpctypes.ResultBlockResults) (ethtypes.
 		}
 
 		for _, attr := range event.Attributes {
-			if bytes.Equal(attr.Key, bAttributeKeyEthereumBloom) {
-				return ethtypes.BytesToBloom(attr.Value), nil
+			if attr.Key == evmtypes.AttributeKeyEthereumBloom {
```

Comment:
any sense whats the reason for this change? I would've assumed the byte comparison to be faster and a deeper level of equality checking 🤷🏽